blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractPipelineCreateRequest.java

```diff
@@ -52,6 +55,16 @@ public AbstractPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
     }
 
     protected ModifiableTopLevelItemGroup getParent() {
-        return Jenkins.getInstance();
+        String organization = getOrganization();
+        ItemGroup parent =  OrganizationResolver.getItemGroup(getOrganization());
+        if(parent == null){
+            throw new ServiceException.BadRequestExpception("Invalid Jenkins organization. " + organization + ". Not found");
+        }
+
+        if(!(parent instanceof ModifiableTopLevelItemGroup)){
```

Comment:
@i386 That might look safe assumption but I am not sure. OrganizationResolver.getItemGroup() is orthogonal to creation. If the assumption of 'organization is always ModifiableItemGroup', then we should remove reference to ItemGroup from OrganizationResolver and from OrganizationImpl. If instanceof check is concern, then maybe we can make it parameterized?